ogr/services/github/project.py

```diff
@@ -362,11 +362,39 @@ def get_tag_from_tag_name(self, tag_name: str) -> Optional[GitTag]:
 
     @if_readonly(return_function=GitProjectReadOnly.pr_create)
     def pr_create(
-        self, title: str, body: str, target_branch: str, source_branch: str
+        self,
+        title: str,
+        body: str,
+        target_branch: str,
+        source_branch: str,
+        fork_username: str = None,
```

Comment:
Can you please update the header of the `pr_create` method also in the abstract class (`ogr/abstract.py`) and other implementations (e.g. `ogr/services/gitlab/project.py`)?

Reply:
done

Reply:
Thanks!